### ISSUE TITLE
tagged release + instructions to build from source?

### ISSUE THREAD

**Issue by boegel** [genometools/genomethreader#1]
It would be nice if the latest release that is available from http://genomethreader.org/download.html (1.7.1) is tagged here, so the sources that correspond to the binary releases can be grabbed easily.

In addition, are there any instructions available on how to compile the latest release from source? Is there a particular reason why you are not providing a source tarball via http://genomethreader.org/download.html ?

**Comment by gordon** [comment]
GenomeThreader is currently build by combining three different repositories (GenomeTools, Vstree, and GenomeThreader) and the build system is not ready for public consumption.

If you really want to dig into building GenomeThreader yourself, you can take a look here:

https://github.com/genometools/gtmetascripts/blob/master/build_all.sh

The source was dropped here so @satta can work on making a Debian package out of Vmatch and GenomeThreader eventually.

But at this point building from source is not supported.

**Comment by gordon** [comment]
I went ahead and tagged the source, though. This is the source code version that was used to build GenomeThreader 1.7.1.

**Comment by Neato-Nick** [comment]
Tagging along with this -

My HPC is stuck on GLIBC 2.17. Running genomethreader, I get errors requiring GLIBC 2.27.

```
/abs/path/to/gth-1.7.2-Linux_x86_64-64bit/bin/gth: /usr/lib64/libc.so.6: version `GLIBC_2.27' not found (required by /abs/path/to/gth-1.7.2-Linux_x86_64-64bit/bin/gth)
```

For other tools, I have solved this by compiling from source myself on our infrastucture

**Comment by gordon** [comment]
Did you try GenomeThreader 1.7.1?

**Comment by Neato-Nick** [comment]
@gordon good idea, just tried that. 1.7.1 looks like it should work! I haven't run test cases with it but at least I can run the executable without the lib version error.

Thanks!

**Comment by gordon** [comment]
If the `1.7.1` binary works then just use that, there didn't really change anything.

**Comment by Neato-Nick** [comment]
@gordon Many thanks, it's running perfectly. If I had been on my best behavior and opened my own issue for the error, I would close it right now.
